src/nvim/tui/tui.c

```diff
@@ -1009,17 +1012,51 @@ static void tui_mouse_on(UI *ui)
 {
   TUIData *data = ui->data;
   if (!data->mouse_enabled) {
+#ifdef WIN32
+    const char *term = os_getenv("TERM");
+    bool toggle_uv_vterm_state =
+      term && (terminfo_is_term_family(term, "conemu")
```

Comment:
is it possible that `term == NULL` but `os_has_vti_working` returns true?

Reply:
After merging #11808, I would like to rewrite the condition using `uv_tty_get_vterm_state()`.